R/fwrite.R

```diff
@@ -38,7 +38,7 @@ fwrite = function(x, file="", append=FALSE, quote="auto",
   # validate arguments
   if (is.matrix(x)) { # coerce to data.table if input object is matrix
     messagef("x being coerced from class: matrix to data.table")
-    x = as.data.table(x)
+    x = if (row.names && !is.null(rownames(x))) { row.names = FALSE; as.data.table(x, keep.rownames="") } else as.data.table(x)
```

Comment:
We normally add a link to the issue number as a comment on lines like this so that the background and further detail can be easily found in future.
@MichaelChirico It shouldn't have been approved because of that. Contributing guide item 1 contains "The test's comment should contain #<issue number> so we can quickly find the issue in future." which this PR doesn't. We could change the guide to add more and clarify the comment standards required, but there is no comment at all on either code line or tests.

Reply:
Guide is pretty clear to me on this point. Might even use a "the test's comment must contain" to enforce it. I guess, I smuggled this one through by extending older test cases instead of adding a new test number.